apps/vanilla/src/global/main.css

```
@@ -0,0 +1,381 @@
+}
+
+.sort-control {
+	/* Remove default arrow */
+	appearance: none;
+	background: var(--shadow-color-tiny);
+	color: var(--main-text-color);
+	padding: var(--gap-small) var(--gap-regular);
+	border: none;
+	cursor: pointer;
+}
+
+.sort-control:invalid {
+	color: var(--shadow-color-large);
+}
+
+.sort-control:focus,
+.search-control:focus {
+	outline: var(--outline-tiny) solid var(--main-text-color);
+}
+
+.show-comics__label {
+	/* Set height like this violates the Wiki best practices but this is for easier to calculate layout later */
+	height: var(--height-sub-input);
+	display: flex;
+	align-items: center;
+	justify-content: flex-start;
+	gap: var(--gap-regular);
+	cursor: pointer;
+}
+
+.show-comics-control {
+	display: none;
+}
+
+.show-comics-control + .show-comics__box {
+	--check-box-size: 20px;
+
+	display: grid;
+	place-items: center;
+	width: var(--check-box-size);
+	height: var(--check-box-size);
+	border: 2px solid var(--shadow-color-medium);
+	border-radius: var(--border-radius-tiny);
+	cursor: pointer;
+}
+
+.show-comics-control:checked + .show-comics__box {
+	border: 2px solid var(--primary-color);
+}
+
+.show-comics-control + .show-comics__box::after {
+	--box-dimension: 10px;
+
+	display: none;
+	content: "";
+	width: var(--box-dimension);
+	height: var(--box-dimension);
+	background-color: var(--primary-dark-color);
+	border-radius: calc(var(--border-radius-tiny) - 1px);
+}
+
+.show-comics-control:checked + .show-comics__box::after {
+	display: block;
+}
+
+.show-comics__text {
+	line-height: 1;
+}
+
+/* Footer styles */
+
+.main__footer {
+	display: flex;
+	align-items: start;
+	justify-content: space-between;
+	width: 100%;
+	height: var(--height-main-footer);
+	padding-top: var(--gap-medium);
+	border-top: 1px solid var(--shadow-color-regular);
+}
+
+.main__footer-more {
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	gap: var(--gap-medium);
+}
+
+.main__footer-more-info {
+	display: flex;
+	align-items: center;
+	gap: var(--gap-small);
+}
+
+.main__footer-more-info-copyright {
+	font-size: var(--font-size-medium);
+	line-height: 1.6;
+}
+
+.main__footer-more-github {
+	display: block;
+	color: var(--main-text-color);
+	text-decoration-line: underline;
+	border-bottom: none;
+	cursor: pointer;
+}
+
+.go-to-top {
+	font-size: var(--font-size-medium);
+	line-height: 1.2;
+}
+
+.go-to-top__image {
+	width: var(--icon-size-small);
+}
+
+/*
+  RESPONSIVE SECTION
+  Use em to avoid unexpected behaviors based on this article -> https://zellwk.com/blog/media-query-units/
+  Change px to em -> https://www.w3schools.com/tags/ref_pxtoemconversion.asp
+  In the design we have breakpoints 1280px, 786px and 360px
+  I add 2 more breakpoints: 1024px and 640px to make the layout more flexible but don't break the rules of the design
+  Additional breakpoints based on Tailwindcss -> Check it here: https://v1.tailwindcss.com/docs/breakpoints
+*/
+
+/* 1280px */
+@media only screen and (max-width: 80em) {
```

```suggestion
@media only screen and (max-width: 1280px) {
```
Let's use pixels and leave the relative units to the fonts.

---

Thanks bro!
Please take a look at this [article](https://zellwk.com/blog/media-query-units/) able px, em and rem, px is not going to work well on Safari.

---

https://developer.mozilla.org/en-US/docs/Web/CSS/max-width#browser_compatibility
https://developer.mozilla.org/en-US/docs/Web/CSS/@media#specifications
https://developer.mozilla.org/en-US/docs/Web/CSS/@media/width#browser_compatibility

Judging by the documentation: everything should work.

https://wiki.saritasa.rocks/frontend/languages/css#be-careful-with-font-related-units
In addition, we have a rule on WIKI for relative units of measurement

---

Thanks bro! I am going to use px in media query from now